.gitignore

```diff
@@ -1,2 +1 @@
 /target
-Cargo.lock
```

Comment:
`Cargo.lock` should stay in the gitignore because souvlaki is a library, not a binary